src/luascript.cpp

```diff
@@ -11550,6 +11551,18 @@ int32_t LuaScriptInterface::luaItemTypeGetDecayId(lua_State* L)
 	return 1;
 }
 
+int32_t LuaScriptInterface::luaItemTypeGetRequiredLevel(lua_State* L)
+{
+	// itemType:getRequireLevel()
+	const ItemType* itemType = getUserdata<const ItemType>(L, 1);
+	if (itemType) {
+		lua_pushnumber(L, itemType->minReqLevel);
+	}else {
```

Comment:
Dude, really? Wtf is your problem hopping on my PR's like that? So cry about it, one space is missing, I just messed with git for an hr trying to squash and make everything right, and you come in with no time nedeed to mess it up again. Not happy in life and got to try to bring others down too or something?